packages/block-editor/src/components/block-draggable/style.native.scss

```diff
@@ -0,0 +1,21 @@
+.draggable-wrapper__container {
+	flex: 1;
+}
+
+.draggable-chip__container {
+	flex-direction: row;
+	padding: 16px;
+	background-color: #f7f7f7;
```

Comment:
I feel like this could match `$gray-0` no?

Reply:
Good catch! Yeah, I rather use a predefined color so I'll use `$gray-0`, thanks for the suggestion 🙇 .

Reply:
Now that you mention that, I noticed that I forgot to add the dark mode styles 😅, I'll add them as soon as possible.

Reply:
>I noticed that I forgot to add the dark mode styles 😅, I'll add them as soon as possible.

That's true! 😄 thanks!